src/wordcab_transcribe/services/asr_service.py

```diff
@@ -776,7 +776,10 @@ async def remote_diarization(
         data: DiarizationRequest,
     ) -> DiarizationOutput:
         """Remote diarization method."""
-        async with aiohttp.ClientSession() as session:
+        # Set the timeout to 12 hours for several hours long audio files
+        verylongtimeout = ClientTimeout(total=12 * 60 * 60)
+
+        async with aiohttp.ClientSession(timeout=verylongtimeout) as session:
```

Comment:
I think that @aleksandr-smechov is only asking to make the timeout configurable. As far as I understand, it's as simple as this?

```suggestion
        timeout = os.getenv("REQUEST_TIMEOUT", 300)
        async with aiohttp.ClientSession(timeout=timeout) as session:
```

Of course, as part of the PR, a change should be done in the `.env` file as well. Also, `os` should be imported in the file.